config/providers.cfg

```diff
@@ -1,3 +1,28 @@
+# Creates Magpie services with specified configuration values on application startup.
+#
+#   - Default location is 'magpie/config/providers.cfg'.
+#   - Directory location can be overridden with 'MAGPIE_CONFIG_DIR' (using filename 'providers.cfg')
+#     or can be explicitly overridden with 'MAGPIE_PROVIDERS_CONFIG_PATH' pointing to a valid file.
+#   - If 'MAGPIE_PROVIDERS_CONFIG_PATH' points to an existing directory, the application will attempt
+#     to process all '.cfg' files found under it one-by-one as separate 'providers' configurations.
+#
+# Parameters
+#   url:        private URL of the service to be created
+#   title:      pretty name of the service (real name is the section key)
+#   public:     parameter passed down to Phoenix for service registration
+#   c4i:        parameter passed down to Phoenix for service registration
+#   type:       service type to use for creation, must be one of the known Magpie service types
```

Comment:
Can add reference to where are declared the list of "known Magpie service types"?

Reply:
Done.